pom.xml

```diff
@@ -714,7 +714,7 @@
       <dependency>
         <groupId>org.scalacheck</groupId>
         <artifactId>scalacheck_${scala.binary.version}</artifactId>
-        <version>1.12.5</version> <!-- 1.13.0 appears incompatible with scalatest 2.2.6 -->
+        <version>1.13.4</version>
```

Comment:
Yes, I did use maven for the entire local built and sbt for this particular `DAGSchedulerSuite` (that it all started from actually).

I've got little experience with Spark test suite and hence I'm relying on Jenkins to do the heavy-lifting and see how things may have changed since the comment.